models/pancreas_ct_dints_segmentation/configs/search.yaml

```diff
@@ -85,7 +85,7 @@ transform_train:
     - label4crop
   - _target_: Lambdad
     keys: label4crop
-    func: "$lambda x, s=@output_classes: np.concatenate(tuple([ndimage.binary_dilation((x==_k).astype(x.dtype), iterations=48).astype(x.dtype) for _k in range(s)]), axis=0)"
+    func: "$lambda x, s=@output_classes: np.concatenate(tuple([ndimage.binary_dilation((x==_k).astype(x.dtype), iterations=48).astype(float) for _k in range(s)]), axis=0)"
```

Comment:
Please also update the metadata, change monai dep to 0.9.1, and the package version to 0.1.1